plugins/jira-dashboard/src/components/JiraTable/JiraTable.tsx

```diff
@@ -20,31 +26,49 @@ export const JiraTable = ({ tableContent }: Props) => {
   }
   const nbrOfIssues = tableContent?.issues?.length ?? 0;
 
+  const filters: TableFilter[] = showFilters
+    ? [
+        {
+          column: 'Status',
+          type: 'multiple-select',
+        },
+        {
+          column: 'Priority',
+          type: 'multiple-select',
+        },
+      ]
+    : [];
+
+  const title = (
+    <Typography component="div" variant="h5" data-testid="table-header">
+      {`${capitalize(tableContent.name)} (${nbrOfIssues})`}
+    </Typography>
+  );
+
   return (
-    <Table
-      title={
-        <Typography component="div" variant="h5" data-testid="table-header">
-          {`${capitalize(tableContent.name)} (${nbrOfIssues})`}
-        </Typography>
-      }
-      options={{
-        paging: false,
-        padding: 'dense',
-        search: true,
-      }}
-      data={tableContent.issues || []}
-      columns={columns}
-      emptyContent={
-        <Typography display="flex" justifyContent="center" pt={30}>
-          No issues found&nbsp;
-        </Typography>
-      }
-      style={{
-        height: '500px',
-        padding: '20px',
-        overflowY: 'auto',
-        width: '100%',
-      }}
-    />
+    <InfoCard title={title}>
+      <Table
+        options={{
+          paging: false,
+          padding: 'dense',
+          search: true,
+        }}
+        filters={filters}
+        data={tableContent.issues || []}
+        columns={columns}
+        emptyContent={
+          <Typography display="flex" justifyContent="center" pt={30}>
+            No issues found&nbsp;
+          </Typography>
+        }
+        style={{
+          height: `max-content`,
+          maxHeight: `500px`,
+          overflowY: 'auto',
+          width: '100%',
+          boxShadow: 'none',
```

Comment:
This really looks good now! However, when showFilters is false, there is a lot of empty space in the component. Any way you could fix this?
![Screenshot from 2024-09-05 15-46-35](https://github.com/user-attachments/assets/14457932-c030-4e73-900f-112319e1905e)

Reply:
The InfoCard could be used optionally? When filters are not enabled, the title is displayed within the Table Component, and the box-shadow is none. If filters are enabled, the InfoCard displays the title, and the box-shadow remains.

Reply:
Looks great now :partying_face: